### DISCUSSION TITLE
Error messsage when installing

### DISCUSSION THREAD

**Discussion by yqwang1** [talmolab/sleap#1338] [Help!]
Hi,
I encountered such information when I installed sleap.
My GPU is NVIDIA [Quadro 600], I updated to the lastest version of driver then I think it maximumly support CUDA 8.0
Can I use CUDA 8.0 instead? 
I also saw option to install with CPU only in the instructions, will sleap function the same if using that path of installation?

Thanks in advance.

**Comment by roomrys** [comment]
Hi @yqwang1,

What is the error message you are getting when you try to install SLEAP?

The version of CUDA (cudatoolkit=11.3.1, cudnn=8.2.1) that SLEAP uses is dependent on the version of tensorflow that SLEAP uses (2.6.3). You can find a list of [tested build configuration for cudatoolkit/cudnn/tensorflow](https://www.tensorflow.org/install/source#gpu) on the tensorflow site. It seems tensorflow_gpu=1.4.0 was the last version that was tested against cudatoolkit=8.0. If you install from source, then you can edit the `environment.yaml` file in the root project directory to specify which version of cudatoolkit/cudnn/tensorflow to use.

If you install SLEAP with just the CPU instructions, then you will be unable to use the GPU on your local computer for training. You will still be able to annotate frames on your local computer, but will need to do the training [remotely](https://sleap.ai/guides/remote.html) or on [online](https://sleap.ai/guides/colab.html).

Thanks,
Liezl

**Reply by yqwang1** [reply]
Thank you for your detailed reply, is that mean if I edited the version of cudatoolkit/cudnn/tensorflow, let's say tensorflow_gpu=1.4.0, cudatoolkit=8.0, SLEAP can still work?

**Reply by roomrys** [reply]
Hi @yqwang1,

I don't think the GPU-integration on SLEAP will work with tensorflow_gpu=1.4.0.... Since we are on a version of tensorflow 2.x.x, I would expects some incompatibilities in our code from what is expected in tensorflow 1.x.x. If you want to use SLEAP purely for annotating on that machine, then I would recommend the non-GPU installation (if installing [from source](https://sleap.ai/installation.html#conda-from-source), otherwise the [conda package](https://sleap.ai/installation.html#conda-package) should be fine as well).

Sorry for the bad news,
Liezl
